deploy/dev/docker-compose.yaml

```diff
@@ -8,6 +8,7 @@ services:
       DEBUG: true
       PORT: "8080"
       IS_DEV_DEPLOY: true
+      ADMIN_USERNAMES: "cceatyo-test@arctretest.onmicrosoft.com" # todo
```

Comment:
do we get accounts for this through the onboarding or..?

Reply:
🤔 – setup from sample is entra id..(?) (happy to move this to slack)